Sources/TuistCore/ContentHashing/HashingFilesFilter.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import TSCBasic
+
+internal class HashingFilesFilter {
+    /// an array of filters, which should return if a path should be included in hashing calculations or not.
+    private let filters: [((AbsolutePath) -> Bool)]
+
+    internal init() {
+        filters = [
+            { $0.basename.uppercased() != ".DS_STORE" },
+        ]
+    }
+
+    func callAsFunction(_ path: AbsolutePath) -> Bool {
```

Comment:
I didn't know about `callAsFunction`, good usage of it!!

Reply:
😃